sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_client_async.py

```diff
@@ -987,6 +1031,9 @@ async def settle_messages_async(
 
     async def settle_messages_async(self, delivery_id: Union[int, Tuple[int, int]], outcome: str, **kwargs):
         batchable = kwargs.pop('batchable', None)
+        timeout = kwargs.pop("timeout", 60)
```

Comment:
Default disp timeout of 60 seconds -- currently not customer editable

Reply:
JS default is 6000, and then they allow customer config via their options

Reply:
Also we only will trigger a timeout if we are getting traffic from the service -- this is what the sender did as well, but could have potential issues. (see where `_update_pending_receipts` is called). Any ideas on if there is a better way?